backends-velox/src/main/scala/io/glutenproject/utils/VeloxDllLoaderUbuntu2004.scala

```diff
@@ -0,0 +1,51 @@
+package io.glutenproject.utils
+
+import io.glutenproject.vectorized.JniLibLoader
+
+class VeloxDllLoaderUbuntu2004 extends VeloxDllLoader {
```

Comment:
Does `dll` imply MS's `.dll`? That's not for Linux.

Reply:
Dll means Dynamic link library here.

Reply:
When I implemented centos this week, I will rename `dll` to `Shared libraries`.  Do you think it is okay？ https://en.wikipedia.org/wiki/Library_(computing)#Shared_libraries